src/Microsoft.Health.Fhir.Core/Features/Operations/Export/CreateExportRequestHandler.cs

```diff
@@ -122,5 +106,72 @@ public async Task<CreateExportResponse> Handle(CreateExportRequest request, Canc
 
             return new CreateExportResponse(outcome.JobRecord.Id);
         }
+
+        /// <summary>
+        /// Parses the _typeFilter parameter from a string into a list of <see cref="ExportJobFilter"/> objects.
+        /// </summary>
+        /// <param name="filterString">The _typeFilter parameter input.</param>
+        /// <returns>A list of <see cref="ExportJobFilter"/></returns>
+        private IList<ExportJobFilter> ParseFilter(string filterString)
+        {
+            var filters = new List<ExportJobFilter>();
+
+            try
+            {
+                if (!string.IsNullOrWhiteSpace(filterString))
+                {
+                    var filterArray = filterString.Split(",");
```

Comment:
Is it possible that one of the `_typefilter` queries could contain a parameter that contains a comma separated list of values? In that case splitting using `,` would cause incorrect splitting.

Reply:
This is a good point, but I'm unsure how to handle it. The comma is specified by the [spec](https://hl7.org/fhir/uv/bulkdata/export/index.html#example-request-with-_typefilter). I could make the parsing guess at what roll the comma has based on the existence of other characters (specifically an equals sign). I'll see what I can do.

Reply:
I would suggest what in that case inside comma need to be escaped.
Also just to note, there is SplitByOrSeparator() extension on top of String type, would be better to use it rather than custom split function.

Reply:
Bumping up this thread to see whether we are planning to handle this @LTA-Thinking

Reply:
Thanks for the reminder, I'll take a look.